Atomic/scan.py

```diff
@@ -21,6 +25,7 @@ def cli(subparser):
     scanp.add_argument("--scan_type", default=None, help=_("define the intended scan type"))
     scanp.add_argument("--list", action='store_true', default=False, help=_("List available scanners"))
     scanp.add_argument("--scanner_args", default=None, help=_("Specify arguments to be passed to the scanner"))
+    scanp.add_argument("--harden", action='store_true', default=False, help=_("Create hardened image after scan"))
```

Comment:
Maybe we can say that scanners can optionally support `--harden` if they wish. Scanners would declare support of automated hardening in their configuration file `/etc/atomic.d/[scanner]`. Atomic could then check if the scanner supports this option. If not, it will print an error message and exit.